### PR TITLE
version 5.9.0

### PR DESCRIPTION
**Pull request by Mastermindzh**

### CODE REVIEW

**Review by Mar0xy — APPROVED** [review]

File: com.mastermindzh.tidal-hifi.metainfo.xml [hunk 1]
```diff
@@ -42,10 +42,22 @@
   </screenshots>
   <content_rating type="oars-1.1" />
   <releases>
+    <release version="5.9.0" date="2024-02-12">
+      <description>
+        <p>Added the ability to hide the current song from the discord activity and display a custom
+          text instead</p>
+        <p>Added the ability to customize the text that is shown when no song is playing</p>
+        <p>Discord now reacts to pausing/unpausing events</p>
+        <p>Refactored media info updates so it only updates the required info</p>
```
Comment (Mar0xy) [inline com.mastermindzh.tidal-hifi.metainfo.xml:51]: This tag is broken

File: com.mastermindzh.tidal-hifi.metainfo.xml [hunk 2]
```diff
@@ -116,7 +130,7 @@
     <release version="5.0.0" date="2023-04-22">
       <description>
         <p>Adblocking support thanks to @thanasistrisp.</p>
-        <p> Replaced "muting artists" with a full implementation of an Adblock mechanism<br /> >
+        <p> Replaced "muting artists" with a full implementation of an Adblock mechanism<br />
```
Comment (Mar0xy) [inline com.mastermindzh.tidal-hifi.metainfo.xml]: metainfo check is very picky and doesn't like br tags

File: com.mastermindzh.tidal-hifi.metainfo.xml (hunk 1 above)
Comment (Mar0xy) [reply com.mastermindzh.tidal-hifi.metainfo.xml:51]: Woops code view messed up this tag

Comment (Mastermindzh) [reply com.mastermindzh.tidal-hifi.metainfo.xml:51]: I don't see what is broken here?

File: com.mastermindzh.tidal-hifi.metainfo.xml (hunk 2 above)
Comment (Mastermindzh) [reply com.mastermindzh.tidal-hifi.metainfo.xml]: Yeah, figured...
Weird thing is that it was already in the code: https://github.com/flathub/com.mastermindzh.tidal-hifi/blob/master/com.mastermindzh.tidal-hifi.metainfo.xml#L119

Comment (Mar0xy) [reply com.mastermindzh.tidal-hifi.metainfo.xml]: Might have been due to recent changes that it no longer works :shrug:

Comment (Mastermindzh) [reply com.mastermindzh.tidal-hifi.metainfo.xml]: we'll see, I removed it... let's hope it builds :)

**Comment by Mastermindzh** [comment]
I tested it, seems to work fine 😄 

Now we wait for it to get the signal and let us merge I guess.

**Pull request merged**